docs/getting_started.rst

```diff
@@ -1,2 +1,136 @@
+            httpProviderUrl: 'https://mainnet.infura.io/v3/[INFURA_PRODUCT_ID]',
+            privateKey
+        },
+        42: {
+            polymathRegistryAddress: '0x5b215a7d39ee305ad28da29bf2f0425c6c2a00b3',
+            httpProviderUrl: 'https://kovan.infura.io/v3/[INFURA_PRODUCT_ID]',
+            privateKey
+        },
+    };
+    const config = networkConfigs[networkId];
+    const polyClient = new Polymath();
+    await polyClient.connect(config);
+
+
+``PolyClient.connect()`` accepts three parameters:
+
+- ``PolymathRegistryAddress``, which is the address of the main registry that keeps track of core components of the system, including the addresses of other registries. 
+- ``httpProviderUrl``, which is an Ethereum network client that enables you to view and interact with the blockchain, via HTTP. That node can be a local ``geth`` or ``parity`` network client, or a 3rd-party provider such a ``Infura``.
+- ``privateKey`` this is the private key used to sign transactions sent from the SDK. 
+
+**On browsers**, some plugins such as Metamask, automatically injects a Web3 instance for your webpage to use. In that case, you can omit passing `httpProviderUrl` and `privateKey`. The SDK will use the injected provider behind the scenes.
+
+Additionally, you can detect the currently selected network using ``browserUtils.getNetworkId()``, which returns network's ID. e.g `1` for mainnet.
+
+Deploying your first security token
+-----------------------------------
+
+There's a multitude of modules and features provided by Polymath smart contracts. Many of them are abstracted by the SDK for simpler usage. In this section, we are going to interact with ``SecurityTokenRegistry`` contract to create our first token.
+
+::
+
+    try {
+        const reservation = await polyClient.reserveSecurityToken({
+            symbol: 'ABC123',
+            name: 'ABC 123 Inc.',
+        });
+        await reservation.run(); // Will run sequentially every transaction required
+
+        const creation = await polyClient.createSecurityToken({
+            symbol: 'ABC123',
+            name: 'ABC 123 Inc.',
+            detailsUrl: 'http://example.com',
+            divisible: true,
+        }).
+        await creation.run();
+
+        // Ta-da!! you've deployed your first security token!
+
+    } catch(error) {
+        // Transaction reverted, transaction has been rejected by user, network issues..etc
+        console.error(error);
+    }
+
+A successful ``reserveSecurityToken()`` call reserves a symbol to the current user address. Reserving a symbol means that no one else will be able to take it, while you complete the necessary steps to deploy the actual token.
+
+Once ``reservation.run()`` resolves, we can proceed with token creation. Method ``createSecurityToken()`` accepts four parameters.
+
+- ``symbol``: the symbol we've just reserved.
+- ``name``: this name will override the name supplied during the reservation.
```

Comment:
This description should be changed to reflect the fact that reservations no longer receive a name